airbyte-cdk/python/airbyte_cdk/sources/declarative/retrievers/simple_retriever.py

```diff
@@ -60,7 +60,7 @@ class SimpleRetriever(Retriever):
     primary_key: Optional[Union[str, List[str], List[List[str]]]]
     _primary_key: str = field(init=False, repr=False, default="")
     paginator: Optional[Paginator] = None
-    stream_slicer: StreamSlicer = field(default_factory=lambda: SinglePartitionRouter(parameters={}))
+    stream_slicer: StreamSlicer = field(default_factory=SinglePartitionRouter(parameters={}))
```

Comment:
Based on [that page](https://docs.pydantic.dev/latest/api/fields/#pydantic.fields.Field), `default_factory` should be a callable that does not take any argument and instantiate any. It feels like it was already like that by this has been changed. Can you explain why?

Reply:
Yes, that seems true — that should still be a lambda